### DISCUSSION TITLE
挂载文件夹下的默认权限是否正常，这些文件是否需要执行权限

### DISCUSSION THREAD

**Discussion by upbeat-backbone-bose** [kkkgo/PaoPaoDNS#101] [问答和帮助] [answered]
![image](https://github.com/kkkgo/PaoPaoDNS/assets/438092/821b0eca-145b-40de-a2ca-cad8fdfe722f)

**Comment by kkkgo** [comment, marked as answer]
为了避免容器和宿主之间用户id不一样导致权限问题，如 https://github.com/kkkgo/PaoPaoDNS/discussions/52
容器启动之后直接777了，可以不用理会。
